core-web/apps/dotcms-ui/src/app/view/components/dot-navigation/components/dot-nav-item/dot-nav-item.component.scss

```diff
@@ -69,15 +69,30 @@
 
 .dot-nav__item {
     align-items: center;
-    display: flex;
-    justify-content: space-between;
+    display: grid;
+    grid-template-columns: 2fr 1fr;
     min-height: $navigation-item-height;
-    padding: 14px 22px 13px 25px; // top and bottom padding to get min 48px height
     position: relative;
     transition: background-color $basic-speed ease;
     width: $navigation-width;
 }
 
+.dot-nav__item-main {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    padding: 14px 0 13px 25px;
```

Comment:
Make use of the spacing variables that you can find here

https://github.com/dotCMS/core/blob/b76b504351f45e51bba3e51f24255fdcf61a9834/core-web/libs/dotcms-scss/shared/_spacing.scss